packages/react-components/src/accordion/docs/Accordion.stories.mdx

```diff
@@ -23,6 +25,28 @@ import { Text } from "@react-components/typography";
 
 ## Accessibility
 
+The content in the Accordion is collapsable. For that reason, it is important to ensure the navigation is not focusing on content that is hidden.
```

Comment:
I am not convince we should mention this. I am not sure what the user should do about this? At least not worded like this.

Reply:
The point here is to ask the user to make sure the keyboard navigation works and makes sense. That if content is collapsed, it shouldn't be focusable and, the other way around, that focusable elements should be focusable when the accordion content is expanded.
I tried on Storybook, with Button it works, but with TextLink for example, it requires to add a `tabindex="0"` not sure if it's by design or an issue with TextLink in the accordion.

Could be:
> Make sure the keyboard navigation behaves accordingly: If the content is collapsed, only the accordion headers should be focusable. Once the content is expanded, focusable elements within those should be part of the keyboard navigation.

Reply:
Could you please provide a short gif illustrating the use case? I have a hard time understanding it.

Is it related to "autoFocus" ?

Reply:
I tried again and I think I tested first with a `TextLink` with no `href` set, which doesn't really make sense. With the `href`, the focus works fine:
![accordion-focus](https://user-images.githubusercontent.com/234418/131399355-77f93d2d-ff37-4cd4-bf3b-500dfc113cb7.gif)
That was the point of that comment but being that Links and Buttons are working fine in those examples, I think I can get rid of that sentence unless we want to keep a mention of the keyboard navigation?

Reply:
I think I would get rid of the sentence. As we discussed earlier, I think the goal of those accessibility blurbs are not to mention all the accessibility concerns which are implementend by the component but rather to mention which actions the user should take or which decisions he might have to make.

Reply:
Then maybe the original copy is enough
> The accordion header that shows and hides the content must be an [heading element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements).

> That heading element should have a level that is appropriate for the information architecture of the page. You can do this by providing an `as` prop to the `<Header>` component.

The added content is things already covered by the component

Reply:
Yeah maybe